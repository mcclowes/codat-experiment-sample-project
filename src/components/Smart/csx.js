import styled from "styled-components/macro";

export const SmartWrapper = styled.div``;

export const SmartContent = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
`;
