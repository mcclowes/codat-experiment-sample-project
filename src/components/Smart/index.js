import SmartContainer from "./SmartContainer";

export default SmartContainer;
