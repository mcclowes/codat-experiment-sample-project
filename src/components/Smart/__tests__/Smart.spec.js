import Smart from "../Smart";
import React from "react";
import { shallow } from "enzyme";

const DEFAULT_PROPS = {};

const renderComponent = (newProps) => {
  const props = {
    ...DEFAULT_PROPS,
    ...newProps,
  };

  return shallow(<Smart {...props} />);
};

let wrapper;

describe("Smart", () => {
  it("renders default component", () => {
    wrapper = renderComponent();

    expect(wrapper).toMatchSnapshot();
  });
});
