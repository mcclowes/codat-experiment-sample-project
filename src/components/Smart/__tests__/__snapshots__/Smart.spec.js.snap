// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Smart renders default component 1`] = `
<csx__SmartWrapper>
  <csx__SmartContent>
    <Button
      doClick={[Function]}
      text="Click me"
    />
    <Padded
      space="p3"
    >
      <csx__Meta
        theme={
          Object {
            "colors": Object {
              "gray": "#aaaaaa",
            },
          }
        }
      >
        Not active
      </csx__Meta>
    </Padded>
  </csx__SmartContent>
</csx__SmartWrapper>
`;
